website/src/components/gallery/ShowcaseCard/index.tsx

```diff
@@ -67,9 +67,9 @@ function ShowcaseCard({ user }: { user: User }): JSX.Element {
   const star = useBaseUrl("/img/Sparkle.svg");
   const fire = useBaseUrl("/img/Fire.svg");
   let azdInitCommand =
-    "azd init -t " + source.replace("https://github.com/", "");
-  if (azdInitCommand.includes("Azure-Samples/")) {
-    azdInitCommand = azdInitCommand.replace("Azure-Samples/", "");
+    "azd init -t " + source.replace("https://github.com/", "").toLowerCase();
+  if (azdInitCommand.includes("azure-samples/")) {
```

Comment:
Consider just making this a case insensitive search, GitHub org names are case-insentive.

Reply:
Missed that part, helps if I read all the lines.  Change looks great, sorry for the noise :-).